tests/integration-tests/tests/nccl/test_nccl/test_nccl/nccl_tests_submit_openmpi.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+#SBATCH --nodes=2
+#SBATCH --exclusive
+#SBATCH --ntasks-per-node=4
+
+module load openmpi
+mpirun \
+-x FI_PROVIDER="efa" \
+-x FI_EFA_USE_DEVICE_RDMA=1 \
+-x RDMAV_FORK_SAFE=1 \
+-x NCCL_ALGO=ring \
+-x LD_LIBRARY_PATH=/shared/openmpi/nccl-2.8.3-1/build/lib/:$LD_LIBRARY_PATH \
+--mca pml ^cm --mca btl tcp,self --mca btl_tcp_if_exclude lo,docker0 --bind-to none \
+/shared/openmpi/nccl-tests-2.0.0/build/all_reduce_perf -b 8 -e 1G -f 2 -g 1 -c 1 -n 100 > /shared/nccl_tests.out
```

Comment:
Nice-to-have: we would want to understand what these options mean and ideally drop a comment about why we have chosen these set of options to run the test. Doc for options is [here](https://github.com/NVIDIA/nccl-tests/blob/master/README.md), but I not sure if we understand why we chose these options.